conda/plugins/manager.py

```diff
@@ -167,19 +166,25 @@ def get_solver_backend(self, name: str = None) -> type[Solver]:
 
         return backend
 
-    def yield_command_hook_actions(self, hook_type: CommandHookTypes, command: str):
+    def apply_pre_commands(self, command: str) -> None:
```

Comment:
```suggestion
    def invoke_pre_commands(self, command: str) -> None:
```

If the docstring uses that term, let's use that verb :D